.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -exu
+
+COMMAND=${1:-"premerge"}
+# Zenko Version
+VERSION=$(cat ../../../VERSION | grep -Po 'VERSION="\K[^"]*')
+# FIXME : waiting for CTST new release
+CTST_VERSION='208e31bc76f7dfb3b4e7357b790c98af86fda96c'
+#`sed 's/.*"cli-testing": ".*#\(.*\)".*/\1/;t;d' ../../../tests/ctst/package.json`
+# Zenko Access Keys
+ZENKO_ACCESS_KEY=$(kubectl get secret end2end-account-zenko -o jsonpath='{.data.AccessKeyId}' | base64 -d)
+ZENKO_SECRET_KEY=$(kubectl get secret end2end-account-zenko -o jsonpath='{.data.SecretAccessKey}' | base64 -d)
+ZENKO_SESSION_TOKEN=$(kubectl get secret end2end-account-zenko -o jsonpath='{.data.SessionToken}' | base64 -d)
+# Extracting kafka host from bacbeat's config
+BACKBEAT_CONFIG_NAME=$(kubectl get secret -l app.kubernetes.io/name=backbeat-config --no-headers -o custom-columns=":metadata.name")
+KAFKA_HOST_PORT=$(kubectl get secret $BACKBEAT_CONFIG_NAME -o jsonpath='{.data.config\.json}' | base64 --decode | jq .kafka.hosts)
+KAFKA_HOST=${KAFKA_HOST_PORT:1:-1}
+# Setting CTST world params
+WORLD_PARAMETERS='{"subdomain":"'${SUBDOMAIN}'","ssl":false,"port":"80","AccountName":"zenko","AccountAccessKey":"'${ZENKO_ACCESS_KEY}'","AccountSecretKey":"'${ZENKO_SECRET_KEY}'","AccountSessionToken":"'${ZENKO_SESSION_TOKEN}'","NotificationDestination":"'${NOTIF_DEST_NAME}'","NotificationDestinationTopic":"'${NOTIF_DEST_TOPIC}'","KafkaHosts":"'${KAFKA_HOST}'"}'
+# Create testing pod
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cli-testing
+  namespace: default
+spec:
+  containers:
+  - name: cli-testing
+    image: registry.scality.com/cli-testing-dev/cli-testing:$CTST_VERSION
+    command: [ "/bin/bash", "-c", "--" ]
+    args: [ "while true; do sleep 30; done;" ]
+    securityContext:
+      allowPrivilegeEscalation: false
+      runAsUser: 0
+    env:
+    - name: TARGET_VERSION
+      value: '$VERSION'
+    volumeMounts:
+    - mountPath: /tmp/package.json
+      name: package
+    - mountPath: /ctst/features
+      name: features
+    - mountPath: /ctst/common
+      name: common
+    - mountPath: /ctst/steps
+      name: steps
+    - mountPath: /ctst/world
+      name: world
+  restartPolicy: Never
+  volumes:
+  - name: package
+    hostPath:
+      path: /Zenko/tests/ctst/package.json
+      type: File
+  - name: features
+    hostPath:
+      path: /Zenko/tests/ctst/features
+      type: Directory
+  - name: common
+    hostPath:
+      path: /Zenko/tests/ctst/common
+      type: Directory
+  - name: steps
+    hostPath:
+      path: /Zenko/tests/ctst/steps
+      type: Directory
+  - name: world
+    hostPath:
+      path: /Zenko/tests/ctst/world
+      type: Directory
+EOF
+# Waiting for pod stabilization
+kubectl wait --for=condition=ready pod cli-testing --timeout 5m
+# Running end2end ctst tests
+kubectl exec -it cli-testing -- ./run $COMMAND $WORLD_PARAMETERS
```

Comment:
You can override the default `--parallel 4` and use a greater number, if you believe this won't add flakiness, it will help you greatly reduce the tests duration.

Reply:
I actually had to reduce it to 2, as there is some flakiness in the tests.
The flakiness is due to a bug, that will be fixed in this PR https://github.com/scality/backbeat/pull/2321

Reply:
It would be great to see if having something like 8 or 16 parallel tests is stable in the CI, if yes, we will greatly reduce the build time for the CTST part :slightly_smiling_face:

Reply:
Yes i completely agree